rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/nats/NatsPublishFactory.kt

```diff
@@ -0,0 +1,15 @@
+package systems.ajax.englishstudytelegrambot.nats
+
+import io.nats.client.Connection
+import org.springframework.stereotype.Component
+
+@Component
+class NatsPublishFactory(val natsConnection: Connection) {
+
+    @Suppress("MagicNumber")
+    fun publish(subject: String, byteArray: ByteArray): Unit =
```

Comment:
why byteArray instead of `GeneratedMessageV3`?

Reply:
my fault, done